ios/Surveys/Sources/Supports/Extensions/SwiftUI/DragGesture+Direction.swift

```diff
@@ -0,0 +1,46 @@
+// swiftlint:disable identifier_name
+//  DragGesture+LeftRight.swift
+//  Surveys
+//
+//  Created by Su Ho on 18/11/2022.
+//  Copyright © 2022 Nimble. All rights reserved.
+//
+
+import SwiftUI
+
+extension DragGesture.Value {
+
+    enum Direction {
+
+        case left
+        case right
+        case up
+        case down
```

Comment:
This is optional. As we don't have any story that will need `up` and `down` directions. I think we could safely remove these 😄 .

Reply:
Actually, we will have the pull to refresh feature on the home screen, so having the down direction would help later. As a result, I prefer to keep all directions :D